wren-ui/src/apollo/server/services/askingService.ts

```diff
@@ -304,14 +335,153 @@ class BackgroundTracker {
   }
 }
 
+class TextBasedAnswerBackgroundTracker {
```

Comment:
Could we move the BackgroundTracker related class under the `server/backgrounds` folder?

Reply:
resolved in [fcd5236](https://github.com/Canner/WrenAI/pull/986/commits/fcd5236dfa598c2982df1e18230c2f91fc61c8ad)